knowledge/data/ir_module_category.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
No need for the encoding any more.

Reply:
Yes for the XML files. It's not needed on Python files in Python 3.